pkg/agent/cache/manager.go

```diff
@@ -61,6 +57,16 @@ type Manager interface {
 	SetAccessControlList(acl *v1.ResourceInstance)
 	GetAccessControlList() *v1.ResourceInstance
 
+	// AccessRequestDefintion cache related methods
```

Comment:
I see handlers populating the cache for these new cached resources, however on initialization the discovery cache job also needs to populate them even in gRPC mode (in case the cache persistent is not enabled)